### DISCUSSION TITLE
Is this project still maintained?

### DISCUSSION THREAD

**Discussion by anater** [hellostealth/stealth#287] [Q&A] [answered]
Hello! Our team is considering tools to enable a custom chat bot experience in our applications. Stealth looks like a great option for us but it's unclear if the project is actively maintained. Are there still contributions being made? Are there plans to address failing CircleCI specs?

**Comment by emorissettegregoire** [comment, marked as answer]
@anater I am a frequent user of Stealth - it is maintained. The founder recently migrated Stealth to Xip Kit. So you have more chance to make sure it is maintained by using this new repo instead: https://github.com/xipkit/xip

**Reply by anater** [reply]
Thank you for the quick response!

**Reply by anater** [reply]
Do you know if the Smooch integration is compatible with Xip Kit?
https://github.com/hellostealth/stealth-smooch

**Reply by emorissettegregoire** [reply]
If I remember right the repo was archived by Stealth.
In case it can help, I recently did an integration with a messaging platform aggregator (using messagebird's omnichannel APIs)
You can have a look here:
--> https://github.com/emorissettegregoire/xip-messagebird
-->https://github.com/emorissettegregoire/stealth-messagebird

**Reply by anater** [reply]
Awesome! I'll take a look. Thanks. You're so helpful!

**Reply by svoop** [reply]
@emorissettegregoire  As for https://github.com/xipkit/xip – I'm not sure how to use a repo which leads to a 404. Do you know anything about what's going on?
